routes/curations.js

```diff
@@ -18,6 +24,19 @@ router.get(
   })
 )
 
+// Get list of components included in a pr
+router.get(
+  '/pr/:pr',
+  asyncMiddleware(async (request, response) => {
+    return curationService.getPrComponents(request.params.pr).then(result => {
```

Comment:
Here add in the URL for the PR so we can avoid people constructing URLs. So the response might look like 

```javascript
{
  pr: 107,
  url: 'https://github.com/....',
  changes: [
    npm/npmjs/-/redie/0.3.0,
    git/github/microsoft/redie/082303205723abc324bd3e
  ]
}
```

That makes the response a bit more stand alone/complete

Reply:
I think that depends on whether or not the base URL is useful elsewhere as well. In the currently disabled `PageCurate` that same URL is hard coded and could be replaced with an API call, but the `/curate` page only shows one component so it's not exactly the same and thus doesn't need the same full dataset either.